ci/pod/opa/allow_get.rego

```diff
@@ -0,0 +1,20 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+package allow_get
+default allow = false
+allow { input.request.method == "GET"}
+status_code = 201 {not allow}
```

Comment:
why do we need a new policy for testing? route related fields don't seem to be checked 🤔

Reply:
> why do we need a new policy for testing?
the existing policies are alread overloaded

> route related fields don't seem to be checked 🤔
I added another commit for this

Reply:
Nice job!